cmd/preview/list.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package preview
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/okteto/okteto/pkg/cmd/login"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+// List lists all the previews
+func List(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Lists all previews",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := login.WithEnvVarIfAvailable(ctx); err != nil {
+				return err
+			}
+
+			err := executeListPreviews(ctx)
+			return err
+
+		},
+	}
+
+	return cmd
+}
+
+func executeListPreviews(ctx context.Context) error {
+	spaces, err := okteto.ListPreviews(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get namespaces: %s", err)
+	}
+	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
+	fmt.Fprintf(w, "Namespace\ttype\tisActive\n")
```

Comment:
```suggestion
	fmt.Fprintf(w, "Name\ttype\tisActive\n")
```
Could you attach an screenshot of the output?

Reply:
<img width="267" alt="Captura de pantalla 2021-07-07 a las 16 50 14" src="https://user-images.githubusercontent.com/25170843/124781179-7e516b00-df43-11eb-8964-e0694ea959ba.png">